WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -60,7 +60,7 @@ public abstract class SendControlViewModel : WasabiDocumentTabViewModel, IWallet
 		private bool _isCustomFee;
 
 		private const string WaitingForHardwareWalletButtonTextString = "Waiting for Hardware Wallet...";
-
+		private readonly long MaxSatoshisSupplyLimit = Constants.MaxSatoshisSupply * 1_000; 
```

Comment:
```suggestion
		private readonly long MaxSatoshisSupplyLimit = Constants.MaxSatoshisSupply / 1_000; 
```
*d'oh* it was in the other side of the inequality, should be divided instead...